testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/suite/suites/Suite7NonGeneric.java

```diff
@@ -50,6 +50,6 @@
                 testOnEnvironment(SinglenodeKerberosHdfsImpersonationCrossRealm.class).withGroups("storage_formats", "cli", "hdfs_impersonation").build(),
                 testOnEnvironment(TwoMixedHives.class).withGroups("two_hives").build(),
                 testOnEnvironment(TwoKerberosHives.class).withGroups("two_hives").build(),
-                testOnEnvironment(SinglenodeLdapBindDn.class).withGroups("ldap").build());
+                testOnEnvironment(SinglenodeLdapBindDn.class).withGroups("ldap").withExcludedGroups("ldap_multiple_binds").build());
```

Comment:
The BindDn env uses a fixed bind user which cannot be used together with bind patterns obviously. So an exclusion was needed.